### PR TITLE
LPS-134222 | LPS-134220

### PR DESCRIPTION
**Pull request by carolmariaabb**
**Related Issues:** https://issues.liferay.com/browse/LPS-134220 and https://issues.liferay.com/browse/LPS-134222

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/validation/FutureDatesDDMValidation.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	service = DDMValidation.class
+)
+public class FutureDatesDDMValidation implements DDMValidation {
+
+	@Override
+	public String getLabel(Locale locale) {
+		return LanguageUtil.get(
+			ResourceBundleUtil.getModuleAndPortalResourceBundle(
+				locale, getClass()),
+			"future-dates");
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/validation/FutureDatesDDMValidation.java]: @brunofariasdeo lembrar de criar a language key

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**